js/AnalyserNode.js

```diff
@@ -130,6 +130,10 @@ module.exports = (jsExport, nativeBinding) => {
     }
 
     set fftSize(value) {
+      if (!(this instanceof AnalyserNode)) {
```

Comment:
Ah yup I was a bit surprised too at first... but actually it makes sens since this is always possible to try to execute anything with the `this` you want in JS,  e.g. this is the test which is made in the idlharness (warning it can hurt you eyes :):

```js
const desc = Object.getOwnPropertyDescriptor(AnalyserNode.prototype, 'fftSize');
desc.set.call({}, 256);
```

> Also, instanceof works with inheritance properly right. So if a user has MyNode extends AnalyserNode these methods are still callable?

Yup it should, which is nice